### PR TITLE
Update OpenMusic to version 1.6

### PR DESCRIPTION
**Pull request by karst**
## What's Changed
* Rock style 4 Artist name typo fix by @karst in https://github.com/OpenRCT2/OpenMusic/pull/51
* ca-ES: Translate Mystic style and OpenRCT2 main theme description. by @J0anJosep in https://github.com/OpenRCT2/OpenMusic/pull/52
* ca-ES: Translate cover add-on by @J0anJosep in https://github.com/OpenRCT2/OpenMusic/pull/53
* Add Progressive style by @karst in https://github.com/OpenRCT2/OpenMusic/pull/54

**Full Changelog**: https://github.com/OpenRCT2/OpenMusic/compare/1.5...v1.6

### CODE REVIEW

File: src/openrct2/object/DefaultObjects.cpp [hunk 1]
```diff
@@ -123,6 +123,7 @@ const std::string_view DefaultSelectedObjects[] = {
     "openrct2.music.prehistoric",
     "openrct2.music.mystic",
     "openrct2.music.rock4",
+    "openrct2.music.progressive"
```
Comment (AaronVanGeffen) [inline src/openrct2/object/DefaultObjects.cpp:126]: Missing trailing comma.

Comment (karst) [reply src/openrct2/object/DefaultObjects.cpp]: Good catch

**Comment by AaronVanGeffen** [comment]
We should probably update the `contributors.md` file in this PR as well, to account for the new music contributor, right?

**Comment by karst** [comment]
Yea, that's probably smart.
The name is Andrew Sinclair (Approaching Nirvana), if you wish to add it. Otherwise I'll add it myself tomorrow.

**Review by AaronVanGeffen — APPROVED** [review]

**Pull request merged**